src/Sarif.Viewer.VisualStudio/Converters/CollectionToVisiblityConverter.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
```

Comment:
I didn't add the header to any of the new files. Is it required? (I can do them all in another PR.)